addons/snackpacks/CfgWeapons.inc

```diff
@@ -0,0 +1,118 @@
+#define CLASSES_WEAPONS         \
+    "SWST_GummiShark"         \
+    , "SWST_TigerThymes"  \
+    , "SWST_SonicBooms"
+
+class CfgWeapons {
+    class CBA_MiscItem_ItemInfo;
+    class ACE_Can_Spirit;
+    class ACE_Sunflower_Seeds;
+    class SWST_Base_Drink : ACE_Can_Spirit {
+        ITEM_META(1);
+        picture = QPATHTOF(data\ui\drink_ui_generic.paa);
+        displayName = "Base Drink Item";
+        descriptionShort = "Drinks are like snacks.<br/>Not that we play with real rations on anyway?";
+        acex_field_rations_consumeText = "Getting sippy of %1...";
+        acex_field_rations_consumeTime = 5;
+        acex_field_rations_thirstQuenched = 5;
+        class ItemInfo: CBA_MiscItem_ItemInfo {
+            mass = 1;
+        };
+    };
+    class SWST_Base_Snack : ACE_Sunflower_Seeds {
+        ITEM_META(1);
+
+        acex_field_rations_consumeText = "Monching %1...";
+        acex_field_rations_consumeTime = 5;
+        acex_field_rations_hungerSatiated = 2;
+
+        displayName = "Base Snack Item";
+        descriptionShort = "Snacks are not meals.<br/>They weigh very little and are just for funsies.";
+        model = QPATHTOF(data\snack_generic.p3d);
+
+        picture = QPATHTOF(data\ui\snack_ui_generic.paa);
+        hiddenSelections[] = {"camo"};
+		hiddenSelectionsTextures[] = {QPATHTOF(data\snackbox\snackbox_generic_co.paa)};
+
+        class ItemInfo: CBA_MiscItem_ItemInfo {
+            mass = 1;
+        };
+    };
+    class SWST_GummiShark : SWST_Base_Snack {
+        ITEM_META(2);
```

Comment:
Since this extends off of a base class, you can just do `SCOPE(2)`. Same for the others that extend from a base.